clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/remote/HotRodStoreResourceDefinition.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.clustering.infinispan.subsystem.remote;
+
+import org.jboss.as.clustering.controller.CapabilityReference;
+import org.jboss.as.clustering.function.Consumers;
+import org.jboss.as.clustering.infinispan.subsystem.InfinispanExtension;
+import org.jboss.as.clustering.infinispan.subsystem.StoreResourceDefinition;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.ModelVersion;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.SimpleAttributeDefinitionBuilder;
+import org.jboss.as.controller.registry.AttributeAccess;
+import org.jboss.as.controller.transform.description.ResourceTransformationDescriptionBuilder;
+import org.jboss.dmr.ModelType;
+import org.wildfly.clustering.infinispan.spi.InfinispanRequirement;
+
+/**
+ * Resource description for the addressable resource:
+ *
+ * /subsystem=infinispan/cache-container=X/cache=Y/store=hotrod
+ *
+ * @author Radoslav Husar
+ */
+public class HotRodStoreResourceDefinition extends StoreResourceDefinition {
+
+    public static final PathElement PATH = pathElement("hotrod");
+
+    public enum Attribute implements org.jboss.as.clustering.controller.Attribute {
+        REMOTE_CACHE_CONTAINER,
```

Comment:
Parameterize using name, type, and capability reference.

Reply:
Fixed.